notebooks/en/semantic_reranking_elasticsearch.ipynb

```diff
@@ -0,0 +1,609 @@
+{
```

Comment:
<p>no need to have a header, just write descriptions otherwise it's less readable </p><p> </p><p>also would be nice to use a HF dataset, maybe upload this dataset to HF and load from there? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>